components/Alert/Alert.stories.tsx

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { Alert } from './Alert';
+
+const meta: Meta<typeof Alert> = {
+  title: 'Components/Alert',
+  component: Alert,
+  tags: ['autodocs'],
+  parameters: {
+    docs: {
+      description: {
+        component: `
+The \`Alert\` component is a dynamic UI element from the Shadcn UI library, designed to display important notifications to users. It combines several subcomponents—\`AlertTitle\`, \`AlertDescription\`, and \`AlertDefault\`—into a cohesive \`AlertNotification\` component. This component is exported as \`<Alert variant={variant} message="Whatever message" />\`, making it easy to use across applications.
+
+### Variants
+
+- **Success**: Indicates successful operations, such as a successful user sign-in.
+- **Error**: Alerts users to errors, like incorrect login information.
+- **Warning**: Reserved for potential issues.
+- **Default**: Used for general information.
+
+### Usage
+
+The \`Alert\` component is primarily used to provide users with visual feedback on the status of their actions, such as confirming successful operations or notifying them of errors. It is integrated with \`react-hot-toast\` to enable popup notifications, enhancing user experience by providing immediate feedback.
+
+\`\`\`jsx
+<Alert variant="success" message="This is a success alert" />
+\`\`\`
+
+### Props
+
+- **variant**: Determines the type of alert, affecting its title and color. Options include 'success', 'error', 'warning', and 'default'.
+- **message**: The content of the alert message, rendered within the \`AlertDescription\`.
+
+### Integration with Toast
+
+The component's notification functionality is powered by \`react-hot-toast\`, a package that handles popup notifications. By wrapping \`toast.custom()\` around \`<Alert variant={} message="" />\`, the component can display alerts as pop-up notifications, providing users with immediate visual feedback.`,
```

Comment:
The spacing looks off with the new lines here. Please remove the new lines
<img width="724" alt="Screenshot 2024-11-14 121807" src="https://github.com/user-attachments/assets/58ee2530-1149-48ae-9046-359bd26875dd">